src/core/tsi/ssl_transport_security.cc

```diff
@@ -1146,6 +1211,17 @@ static tsi_result ssl_handshaker_result_extract_peer(
       &peer->properties[peer->property_count]);
   if (result != TSI_OK) return result;
   peer->property_count++;
+
+  if (ca_cert != nullptr) {
+    result = peer_property_from_x509_subject(
+        ca_cert, &peer->properties[peer->property_count], true);
+    if (result != TSI_OK) {
+      gpr_log(GPR_ERROR, "resut: %d", static_cast<int>(result));
```

Comment:
I'm not sure this should be an error log, since this will occur routinely e.g. if a self-signed cert is used. Could we make it a debug log instead?

Reply:
Resolving because I think we will address this in this thread: https://github.com/grpc/grpc/pull/32215#discussion_r1089523074